README.md

```diff
@@ -508,12 +508,33 @@
     [abstraction](https://en.wikipedia.org/wiki/Abstraction_(software_engineering)), and
     [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
 
-* **What is the difference between a constructor and a method?** 
+* **What is the difference between a constructor and a method?**
     - The name of the constructor is same as that of the class name, whereas the name of the method can be anything.
     - There is no return type of a constructor.
-    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to call it explicitely.
+    - When you make an object of a class, then the constructor of that class will be called automatically. 
+      But for methods, we need to call it explicitely.
     - Constructors can't be inherited but you can call the constructor of the parent class by calling `super()`.
-
+    - Constructor and a method they both run a block of code but the difference is in calling them.
+    - We can call method directly using their name.
+    - Constructor Syntax -         
+              public class SomeClass{
+                SomeClass(parameter_list){ 
+                  ...
+                } 
+              ...
+              }
```

Comment:
Just remove extra space from the starting of every line i.e. remove extra space before `public` and the same goes with other lines also.... Change the class name to `SomeClassName`
Same is for the method part also..

Reply:
Done!